example/utils/GenerateSDFMaterial.js

```diff
@@ -56,9 +56,13 @@ export class GenerateSDFMaterial extends ShaderMaterial {
 					vec3 faceNormal;
 					vec3 barycoord;
 					float side;
+					float rayDist;
 					vec3 outPoint;
 					float dist = bvhClosestPointToPoint( bvh, point.xyz, faceIndices, faceNormal, barycoord, side, outPoint );
 
+					//bool hit = bvhIntersectFirstHit( bvh, point.xyz, vec3(0.0, 0.0, 1.0), faceIndices, faceNormal, barycoord, side, rayDist );
+					//side = ( hit && side > 0.0 ) ? 1.0 : -1.0;
```

Comment:
This hit condition needs to be set to `hit && side > 0 || ! hit` so the sign is positive even when not hitting the mesh. But because the `side` inout variable isn't modified if there is no hit you can just set it like so:

```js
side = 1.0;
bvhIntersectFirstHit( bvh, point.xyz, vec3( 0.0, 0.0, 1.0 ), faceIndices, faceNormal, barycoord, side, rayDist );
```

Reply:
This suggestion works great!